### PR TITLE
docs(docs): docs about updating the docs - INNO-853

### PR DESCRIPTION
**Pull request by kalinchernev**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
- [x] I have checked the dependencies
- [x] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [x] I have provided tests
- [x] I follow the naming guidelines
- [ ] the component supports composition
- [x] there are no hardcoded strings (all content come from the context)
- [x] I have filled the README.md file (at least a few lines)
- [x] My component is listed in the root README
- [x] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview ready!

Built with commit 4ee165503c6de6969e3844f6c23a3a4e292748f6

https://deploy-preview-507--europa-component-library.netlify.com

File: docs/04-documentation.md [hunk 1]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
 ## Prerequisite
 
-If you don't already have a GitHub account, make sure to create one and log in with it.  
-You also have to be part of team [ECL doc editors](https://github.com/orgs/ec-europa/teams/ecl-doc-editors/).
+If you don't already have an account on [GitHub](https://github.com/login), make
+sure to create one and log in with it.
 
-## Step 1: Open documentation file
+You also have to be part of team
+[ECL doc editors](https://github.com/orgs/ec-europa/teams/ecl-doc-editors/). for
```
Comment (raduchiriac) [inline docs/04-documentation.md]: Capital F. `for an invite...`

File: docs/04-documentation.md [hunk 2]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
+<details><summary>_See how to do this step_</summary>
+
+![Edit documentaiton file]({{ path
+'/assets/screenshots/edit-documentation-file.gif' }})
+
+</details>
+
+## Step 4: Edit the documentation
+
+-   Make any modification you wish by using markdown editor. Here is a
```
Comment (raduchiriac) [inline docs/04-documentation.md]: "by using __the__ markdown"

File: docs/04-documentation.md [hunk 3]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
+  [cheatsheet for markdown syntax](https://guides.github.com/features/mastering-markdown/)
+
+-   You can see result directly by switching to the "Preview" tab
```
Comment (raduchiriac) [inline docs/04-documentation.md]: see **the** result directly

File: docs/04-documentation.md [hunk 4]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
+
+<details><summary>_See how to do this step_</summary>
+
+![Preview changes]({{ path '/assets/screenshots/edit-preview-changes.gif' }})
+
+</details>
+
+## Step 5: Save updates to the branch
+
+-   Once you have modified the file, you have to commit changes to save them in
```
Comment (raduchiriac) [inline docs/04-documentation.md]: Better: `Once you made sure you have saved`

File: docs/04-documentation.md [hunk 5]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
+![Save changes to branch]({{ path
+'/assets/screenshots/save-changes-on-branch.gif' }})
+
+</details>
+
+## Step 6: Open a pull request
+
+-   Once make sure you have saved changes on your particular branch, you are ready
+  to proceed on proposing these changes to the main `master` branch which is
+  where you see production documentation.
```
Comment (raduchiriac) [inline docs/04-documentation.md]: see **the** production

File: docs/04-documentation.md [hunk 6]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
+
+<details><summary>_See how to do this step_</summary>
+
+![Open a pull request]({{ path '/assets/screenshots/open-pull-request.gif' }})
+
+</details>
+
+## Step 7: Preview your changes and follow-up
+
+-   When the pull request is open, our `HAL-Patch-INNO` bot will provide you a
```
Comment (raduchiriac) [inline docs/04-documentation.md]: "will <strike>provide</strike> **show** you a preview"

File: docs/04-documentation.md [hunk 7]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
+  preview of the ECL style guide with the updates provided by you.
+
+-   You can review them and make further changes on the branch repeating steps
+  1-5. Once the changes are saved, pull request is automatically updated, no
```
Comment (raduchiriac) [inline docs/04-documentation.md]: "saved, **the** pull request"

File: docs/04-documentation.md [hunk 8]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
+  need to open another pull request.
+
+-   Changes on the pull request update the preview automatically.
```
Comment (raduchiriac) [inline docs/04-documentation.md]: "Changes on the pull request, update**s** the preview automatically."

File: docs/04-documentation.md [hunk 9]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
+<details><summary>_See how to do this step_</summary>
+
+![Preview changes]({{ path '/assets/screenshots/preview-changes.gif' }})
+
+</details>
 
 ## It's done!
-Your pull request will be handled by our team and integrated to master ECL branch.  
+
+Your pull request will be briefly reviewed by our team that there are no linting
```
Comment (raduchiriac) [inline docs/04-documentation.md]: Long phrase, commas:

"Your pull request will be briefly reviewed by our team, **so** that there are no linting issues and if all checks are green, changes will be integrated to **the** master ECL branch in short time.

File: docs/04-documentation.md [hunk 10]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
 
 ### Note: writing convention
 
-We are using writing conventions for both [pull requests](https://ec-europa.github.io/europa-component-library/docs/conventions/git) and [markdown](https://ec-europa.github.io/europa-component-library/docs/conventions/markdown). You can optionally apply these conventions, to facilitate documentation update process.  
-In any case, we will have to apply the conventions on our side before publishing it (so the content may be slightly modified).
+We are using writing conventions for both
+[pull requests](https://ec-europa.github.io/europa-component-library/docs/conventions/git)
+and
+[markdown](https://ec-europa.github.io/europa-component-library/docs/conventions/markdown).
+You can optionally apply these conventions, to facilitate documentation update
```
Comment (raduchiriac) [inline docs/04-documentation.md]: No commas before a verb ", to facilitate"

File: docs/04-documentation.md (hunk 1 above)
Comment (raduchiriac) [reply docs/04-documentation.md]: This page explains how to update **the** documentation directly on GitHub.

File: docs/04-documentation.md [hunk 11]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
+<details><summary>_See how to do this step_</summary>
+
+![Create a new branch for documentation updates]({{ path
+'/assets/screenshots/create-new-branch.gif' }})
+
+</details>
+
+## Step 3: Open documentation file for editing
+
+Ensure that you have switched to a new branch which is a good place to suggest
```
Comment (raduchiriac) [inline docs/04-documentation.md]: "switched to <strike>a</strike> your new (or newly created) branch"

File: docs/04-documentation.md [hunk 12]
```diff
@@ -7,40 +7,152 @@ This page explains how to update documentation directly on GitHub.
...
+<details><summary>_See how to do this step_</summary>
+
+![Save changes to branch]({{ path
+'/assets/screenshots/save-changes-on-branch.gif' }})
+
+</details>
+
+## Step 6: Open a pull request
+
+-   Once make sure you have saved changes on your particular branch, you are ready
```
Comment (raduchiriac) [inline docs/04-documentation.md]: Once **you made** sure

File: docs/04-documentation.md (hunk 4 above)
Comment (raduchiriac) [reply docs/04-documentation.md]: commit **the** changes

**Pull request merged**